pygluu/kubernetes/gui/templates/wizard/couchbase_calculator.html

```diff
@@ -0,0 +1,17 @@
+{{ forms.text(form.number_of_expected_users) }}
```

Comment:
Calculations of resources needed for CB is in alpha. We should clearly indicate that visually to the customer. Please add a warning alpha message.